EPPlus/ExcelCellBase.cs

```diff
@@ -212,8 +228,20 @@ private static string ToR1C1(string part, int row, int col, int rowIncr, int col
         /// <param name="rowIncr"></param>
         /// <param name="colIncr"></param>
         /// <returns></returns>
-        private static string ToAbs(string part, int row, int col, int rowIncr, int colIncr)
+        private static string ToAbs(string part, int row, int col, int rowIncr, int colIncr, bool isNoRange = false)
         {
+            // Allow for static R1C1 style column references or static column range references
+            string pattern = @"^C([0-9]+)$";
+            if (Regex.IsMatch(part, pattern))
+            {
+                Match match = Regex.Matches(part, pattern)[0];
+                var absBit = $"${GetColumnLetter(int.Parse(match.Groups[1].Value))}";

```

Comment:
string interpolation in .NET Core is slow now (optimization in roslyn still waits for merge https://github.com/dotnet/roslyn/pull/22595). Use string Concatenation instead